package.json

```diff
@@ -3,14 +3,24 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "lint": "eslint '**/*.js'",
```

Comment:
source: https://stackoverflow.com/questions/36307581/how-to-configure-package-json-to-run-eslint-script